content/300-guides/050-database/100-developing-with-prisma-migrate/175-squashing-migrations.mdx

```diff
@@ -92,7 +92,13 @@ Then follow these steps, either on your `main` branch or on a newly checked out
 
 1. Delete all contents of the `./prisma/migrations` directory
 
-2. Create a new empty directory in the `./prisma/migrations` directory. In this guide this will be called `squashed_migrations`. Inside this, add a new empty `migration.sql` file.
+2. Create a new empty directory in the `./prisma/migrations` directory. In this guide this will be called `000000000000_squashed_migrations`. Inside this, add a new empty `migration.sql` file.
+
+   <Admonition type="info">
+   
+    We name the migration `000000000000_squashed_migrations` with all the leading zeroes because we want it to be the first migration in the migrations directory. Migrate runs the migrations in the directory in lexicographic (alphabetical) order. This is why it generates migrations with the date and time as a prefix when you use `migrate dev`. You can give the migration another name, as long as it it sorts lower than later migrations, for example `0_squashed` or  or `202207180000_squashed`.
```

Comment:
```suggestion
    We name the migration `000000000000_squashed_migrations` with all the leading zeroes because we want it to be the first migration in the migrations directory. Migrate runs the migrations in the directory in lexicographic (alphabetical) order. This is why it generates migrations with the date and time as a prefix when you use `migrate dev`. You can give the migration another name, as long as it it sorts lower than later migrations, for example `0_squashed` or `202207180000_squashed`.
```